andela/travel_tool_front
# #161875591 User can see the right components on the request page after fi…

#### What does this PR do?
Adds end-to-end test for request page for the scenario when a user logs into the application the first time

#### Description of Task to be completed?
- Add end-to-end test to confirm visiblity of necessary items
- Add fixture to mock data when user has no request
- Add custom command yo authenticate user
- Add `eslint-plugin-cypress` package to specify globals for Cypress


#### How should this be manually tested?
- Open the terminal  
- Run `yarn end2end` to run the test with browser mode
- When redirected to the browser, select `component/Requests/request.spec.js`
- The tests should run and pass
- To run test only in the console, run `yarn end2end:headless` 

#### Any background context you want to provide?
- Ensure both the frontend and backend servers are running

#### What are the relevant pivotal tracker stories?
[#161875591](https://www.pivotaltracker.com/story/show/161875591)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
